component/prometheus/relabel/relabel_test.go

```diff
@@ -0,0 +1,81 @@
+package relabel
```

Comment:
Can we add a benchmark here so we can start tracking the performance of this?

Reply:
Adding an incredibly simple best case benchmark. 146ns with cache, 740ns without cache.

![Screen Shot 2022-10-18 at 3 59 45 PM](https://user-images.githubusercontent.com/3036838/196532039-50e2f4ac-b4b1-479c-9071-690a2106a977.png)